### PR TITLE
Sonar issues

### PR DESCRIPTION
**Pull request by dimaru-coti**
Code refactoring and additional fixes

### CODE REVIEW

File: storagenode/src/main/java/io/coti/storagenode/database/DbConnectorService.java [hunk 1]
```diff
@@ -106,24 +102,10 @@ private void addIndex(String indexName, String objectName, boolean fromColdStora
 
     @Override
     public ClusterGetSettingsResponse getClusterDetails(Set<String> indexes) throws IOException {
-        MainResponse mainResponse = restClient.info(RequestOptions.DEFAULT);
```
Comment (gmesika-coti) [inline storagenode/src/main/java/io/coti/storagenode/database/DbConnectorService.java]: i understand that this is not in use. but maybe its a pre-ready code that was supposed to be used for something... lets not delete it entirely

File: storagenode/src/main/java/io/coti/storagenode/database/DbConnectorService.java [hunk 2]
```diff
@@ -106,24 +102,10 @@ private void addIndex(String indexName, String objectName, boolean fromColdStora
...
-            String searchShardsDetails = getSearchShardsDetails(index);
-            ClusterHealthRequest clusterHealthRequest = new ClusterHealthRequest(index);
-            ClusterHealthResponse response = restClient.cluster().health(clusterHealthRequest, RequestOptions.DEFAULT);
-        }
-
         ClusterGetSettingsRequest clusterGetSettingsRequest = new ClusterGetSettingsRequest();
         return restClient.cluster().getSettings(clusterGetSettingsRequest, RequestOptions.DEFAULT);
     }
 
-    private String getSearchShardsDetails(String index) {
```
Comment (gmesika-coti) [inline storagenode/src/main/java/io/coti/storagenode/database/DbConnectorService.java]: also leave this too

File: storagenode/src/main/java/io/coti/storagenode/services/EntityStorageService.java [hunk 3]
```diff
@@ -45,19 +45,18 @@ private <T extends IPropagatable> T retrieveObjectFromStorage(Hash hash) {
...
     }
 
-    protected boolean verifyRetrievedSingleObject(Hash objectHash, String objectAsJson, boolean fromColdStorage, ElasticSearchData objectType) {
+    protected boolean verifyRetrievedSingleObject(Hash objectHash, String objectAsJson, boolean fromColdStorage) {
 
         if (objectAsJson != null && validateObjectDataIntegrity(objectHash, objectAsJson)) {
             if (!fromColdStorage) {
                 try {
                     log.debug("In the future, the object will be deleted from cold storage");
-                    //    objectService.deleteObjectByHash(objectHash, true, objectType)
```
Comment (gmesika-coti) [inline storagenode/src/main/java/io/coti/storagenode/services/EntityStorageService.java]: dont revert this.

File: trustscore/src/main/java/io/coti/trustscore/http/data/BucketEventResponseData.java [hunk 4]
```diff
@@ -18,6 +18,7 @@
     private String bucketHash;
     private Date startPeriodTime;
     private double calculatedDelta;
+    @SuppressWarnings("java:S1948")
```
Comment (gmesika-coti) [inline trustscore/src/main/java/io/coti/trustscore/http/data/BucketEventResponseData.java]: why this supress?

File: zerospend/src/main/java/io/coti/zerospend/services/DspVoteService.java [hunk 5]
```diff
@@ -1,5 +1,6 @@
 package io.coti.zerospend.services;
 
+import afu.org.checkerframework.checker.oigj.qual.O;
```
Comment (gmesika-coti) [inline zerospend/src/main/java/io/coti/zerospend/services/DspVoteService.java]: this is probably not needed and by mistake

**Review by gmesika-coti — CHANGES_REQUESTED** [review]
generally about the move of the enums. plz check if it is working with older data , meaning running on sablenet. take this commit with cherry pick and run the testing

File: trustscore/src/main/java/io/coti/trustscore/http/data/BucketEventResponseData.java (hunk 4 above)
Comment (dimaru-coti) [reply trustscore/src/main/java/io/coti/trustscore/http/data/BucketEventResponseData.java]: It's suppressing this issue: 
Fields in a Serializable class must themselves be either Serializable or transient even if the class is never explicitly serialized or deserialized.

File: zerospend/src/main/java/io/coti/zerospend/services/DspVoteService.java (hunk 5 above)
Comment (dimaru-coti) [reply zerospend/src/main/java/io/coti/zerospend/services/DspVoteService.java]: Yes, removed

**Pull request merged**